Update accessing-a-sql-database-entities.md

GoFightNguyen opened the pull request:
# Fixed incorrect method indentation
## Summary

Correcting the indentation of the addInstructor function, which is the first code sample under the Updating the Database section.

Commit:
Update accessing-a-sql-database-entities.md

Correcting the indentation of the addInstructor function, which is the first code sample under the Updating the Database section.

dnfclas commented:
Hi __@jasonsnguyen__, I'm your friendly neighborhood .NET Foundation Pull Request Bot (You can call me DNFBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, DNFBOT;

qinezh commented:
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/accessing-a-sql-database-entities?branch=pr-en-us-1106](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/accessing-a-sql-database-entities?branch=pr-en-us-1106)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C3%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610031645391827-1106%5Cworkflow_report.html).

svick commented on docs/fsharp/tutorials/type-providers/accessing-a-sql-database-entities.md:
```diff
@@ -183,17 +183,17 @@ let fullContext = context.DataContext
+  let hireDate = DateTime.Parse(hireDate)
+  let newPerson = new EntityConnection.ServiceTypes.Person(LastName = lastName,
                                                          FirstName = firstName,
                                                          HireDate = nullable hireDate)
```
Nit: I think these two lines should be indented too, so that they stay aligned with the first argument.

jasonsnguyen replied:
I agree with you @svick. I made the change in my fork. Please advise on what I need to do next. This is my first attempt at a pull request.

svick replied:
Now you just wait until someone from Microsoft (read: not me) comes along and either accepts the PR or tells you what else to improve in the PR (but with such a small PR, I think there's not much room for that).

Commit:
Update accessing-a-sql-database-entities.md

Indenting two parameters so they stay in-line with LastName in the addInstructor method definition

mairaw commented:
The changes look good. I'll wait for the CI builds to finish before merging this. Thanks @jasonsnguyen!

cartermp reviewed (approved):
Looks good to me.  If it looks fine on staging (I imagine it will), I'll merge it.  Thanks, @jasonsnguyen!

cartermp commented:
Heh, @mairaw got to it first

qinezh commented:
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/accessing-a-sql-database-entities?branch=pr-en-us-1106](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/accessing-a-sql-database-entities?branch=pr-en-us-1106)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C3%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610031959160433-1106%5Cworkflow_report.html).
